payroll-processor-client/src/app/employee/employee-list/state/employee-list.model.ts

```diff
@@ -0,0 +1,39 @@
+export interface Employee {
+  id: string;
```

Comment:
We could make some of these `readonly`, like `id` and `version`.

Reply:
i agree, employmentStartedOn as well

Reply:
Sure firstName and email too, since i'm thinking in a relational way email would be a primary key of some sort. And in most cases, last name may change more likely than first name